### DISCUSSION TITLE
Brownie Compile Error -- Lessons 5 & 6

### DISCUSSION THREAD

**Discussion by hhopkins95** [smartcontractkit/full-blockchain-solidity-course-py#42] [Q&A] [answered]
I'm receiving the following error when running 'brownie compile' near the start of lesson 6. I was running into the same issue in lesson 5 as well:

![image](https://user-images.githubusercontent.com/81984600/133805965-024bfbd8-95ba-4b9a-b772-88489c6c9840.png)

I get a lint error in the Lottery.sol file as well that reads "Source "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol" not found: File import callback not supported"

![image](https://user-images.githubusercontent.com/81984600/133806314-d10680bb-1ea1-48ae-b80b-341932d9b0c5.png)

I've tried different compiler versions with no luck. 

I'm not sure what could be wrong with my file structure or config file either -- 
![image](https://user-images.githubusercontent.com/81984600/133806790-c9d42143-213c-4218-a98d-b64f4513a973.png)

Any help would be very appreciated!

**Comment by PatrickAlphaC** [comment]
Thanks for making this question!

1. You can ignore `"Source "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol" not found: File import callback not supported"`. The linter doesn't really understand brownie... yet.
2.  Could you copy paste the code from those files instead of images?
3. What happens when you use `pragma solidity ^0.6.6;` in your contract instead of what you currently have?

**Reply by hhopkins95** [reply]
Hi Patrick -- thanks for the response. I'm really enjoying the course so far!

Looks like the issue was a simple misspelling on my part. David helped me out below

**Comment by davidyuill27** [comment, marked as answer]
Your brownie-config.yaml is spelled incorrectly. Let us know if that fixes your compiler issue. I had issues with imports as well. I had to npm install the dependencies and put them in a node_modules folder. This allowed the compiler to see actual real issues when coding.

**Reply by davidyuill27** [reply]
I can give more info if needed.

**Reply by hhopkins95** [reply]
I feel very stupid haha. Thank you

**Reply by davidyuill27** [reply]
Haha no worries, easily overlooked!
